### DISCUSSION TITLE
Still frames and keyframes for the Pixelorama animation system

### DISCUSSION THREAD

**Discussion by QuetzalcoutlDev** [Orama-Interactive/Pixelorama#1028] [Ideas]
Well, I've been using Pixelorama for a long time, and I love it

But there is one thing that I feel is missing when animating and that is still frames

What do I mean? Well, I feel that to animate you only have to use keyframes, it makes making changes a little tedious.

This sketch illustrates a little more what I'm saying in case I didn't understand it:

![1111](https://github.com/Orama-Interactive/Pixelorama/assets/102568894/9b927a73-bbf5-44ac-bf01-5d5c2768b9ca)

I mean, what I'm saying is that it would be very necessary to give Pixelorama the option to use still frames or key frames when animating, the truth is that it would help a lot

Since sometimes I don't make changes to a layer and when I do, I have to go from frame to frame changing it

Of course, not if this has already been proposed before

**Comment by OverloadedOrama** [comment]
Are you referring to a way to have multiple frames be the same image? Because that is already implemented in the form of linked cels. Linked cels share the same image data, and if you make changed to one frame, the rest of the linked frames automatically change. https://www.oramainteractive.com/Pixelorama-Docs/concepts/cel#linked-cels

**Reply by QuetzalcoutlDev** [reply]
@OverloadedOrama Wow, so I should have read a little more to understand why that worked, I thought there was no way to do what it explained
